main.cpp

```diff
@@ -1,6 +1,17 @@
 #include <iostream>
 
+void foo()
+{
+    std::cout << "Foo 1" << std::endl;
```

Comment:
Use loop to avoid duplication

Reply:
Not needed because exactly 5 iterations is required.

Reply:
Anyway use a loop

Reply:
fixed